doc/vim-go.txt

```diff
@@ -1085,6 +1085,9 @@ if            "inside a function", select contents of a function,
               excluding the function definition and the closing bracket. This
               text-object also supports literal functions
 
+                                               *go-v_ac* *go-ac*
+ac            "a comment", select contents of the current comment block.
+
```

Comment:
Is there way we could support `ic` as well? This could be handy if we want to remove the comment with leaving it intact. For example `// this is a comment` could be deleted via `dic`, so it would end up with `//`